ts/src/grpc.ts

```diff
@@ -0,0 +1,200 @@
+import * as jspb from "google-protobuf";
+import {BrowserHeaders} from "browser-headers";
+import {ChunkParser, Chunk, ChunkType} from "./ChunkParser";
+import {Transport,DefaultTransportFactory} from "./transports/Transport";
+
+export {
+  BrowserHeaders
+};
+
+export namespace grpc {
+
+  export interface ProtobufMessageClass<T extends jspb.Message> {
+    new(): T;
+    deserializeBinary(bytes: Uint8Array): T;
+  }
+
+  export enum Code {
+    OK = 0,
```

Comment:
how do docs look like in TS?
Can we add strings for these so the IDE auto completes them?